tools/README.md

```diff
@@ -0,0 +1,8 @@
+### Supported tools
```

Comment:
Add the following sentence
"The JSON RPC relay serves as an interface to the Hedera network for ethereum developer tools that utilize the implemented [JSON RPC APIs](../../docs/rpc-api.md)
The following development tools have been tested and the extent of their coverage is noted below"